CONTRIBUTING.md

```diff
@@ -49,28 +49,28 @@ We try to get to pull requests in a very timely way so they don't languish. Noth
 * branch: `git checkout -b feature_branch_name`
 * add unit-test
 * commit: `git commit -m 'failing test'` (you can do this more often)
-* write code 
-* test: `./skulpt.py test` and `./skulpt.py dist`
+* write code
+* test: `npm test` and `npm run build`
 * commit: `git commit -m 'implement fix'` (you can do this more often)
 * push: `git push -u origin feature_branch_name`
-* pull-request: (on github) 
+* pull-request: (on github)
 
 
 
 
 ## Coding Style and Conventions
 
 Here are some coding conventions to keep in mind:
- 
-* ``Sk.ffi.remapToJs`` and ``Sk.ffi.remapToPy`` are your friends.  They take care of the details 
+
+* ``Sk.ffi.remapToJs`` and ``Sk.ffi.remapToPy`` are your friends.  They take care of the details
 of going from a Python type to a Javascript type and vice versa.  They are smart enough to work
-with common types and even work well recursively converting containers.  ``Sk.ffi.remapToJs`` is 
+with common types and even work well recursively converting containers.  ``Sk.ffi.remapToJs`` is
 **definitely preferred** over  ``foo.v``  
 * Use the ``pyCheckArgs`` function at the beginning of anything that will be exposed to a Python programmer.
 * Check the types of arguments when you know what they must be.
 * Explicitly return ``Sk.builtin.null.null$`` for functions and methds that should return ``None``
```

Comment:
I think, this should be `Sk.builtin.none.none$`. Not related to the PR, but maybe you can just change it anyways.